community/modules/scheduler/schedmd-slurm-gcp-v5-controller/main.tf

```diff
@@ -56,7 +56,7 @@ data "google_compute_default_service_account" "default" {
 }
 
 module "slurm_controller_instance" {
-  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_controller_instance?ref=5.4.1"
+  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_controller_instance?ref=5.5.0"
```

Comment:
I believe default images also need to be updated. [ref](https://github.com/SchedMD/slurm-gcp/blob/5.5.0/docs/images.md#published-image-family)

The default images are specified in the variables.tf file for each module.